internal/component/loki/process/stages/multiline.go

```diff
@@ -190,8 +190,9 @@ func (m *multilineStage) flush(out chan Entry, s *multilineState) {
 		Entry: loki.Entry{
 			Labels: s.startLineEntry.Entry.Labels.Clone(),
 			Entry: logproto.Entry{
-				Timestamp: s.startLineEntry.Entry.Entry.Timestamp,
-				Line:      s.buffer.String(),
+				Timestamp:          s.startLineEntry.Entry.Entry.Timestamp,
+				Line:               s.buffer.String(),
+				StructuredMetadata: s.startLineEntry.StructuredMetadata,
```

Comment:
```suggestion
				StructuredMetadata: s.startLineEntry.Entry.Entry.StructuredMetadata,
```

I agree that `Entry.Entry` looks weird, but I'd prefer to keep it this way. It's more consistent with the rest of the code and it can also prevent ambiguity about what struct field is being used.